docs/src/templates/pages/styleguide/ui-components/order-card.hbs

```diff
@@ -0,0 +1,55 @@
+---
+title: Previous Order Card
+section-title: UI Components
+docs: true
+navgroup: styleguide
+navsub: ui-components
+navactive: order-card
+layout: components
+---
+<p><em>The <code>c-orderCard</code> component is an optional mixin within Fozzie — if you'd like to use it in your project you can include it by adding <code>@include orderCard();</code> into your SCSS dependencies file.</em></p>
+<h2 class="sg-sectionHeading">Order Card</h2>
+
+{{#>demo-block demoVisualCssClass="demo-visual--spacingTopAndBottom"}}
+<div class="l-innerContainer">
+    <div class="c-overflowCarousel--withBlockElements">
+
+{{> je-orderCard
+        optionalClassNames="c-overflowCarousel-item"
+        restaurantName="Pizza Loco"
+        restaurantLink="#"
+        orderAgainLinkTitle="Order this meal again"
+        restaurantImageUrl="/assets/img/examples/cuisines-widget.jpg"
+        orderDate="01/12/2018"
+        orderSummary="1 x Sushi Salmon, 3 x Tuna Sashimi, 1 x Coconut Water"
+        showMoreText="+ 3 more"
+        orderTotal="£45.00"
```

Comment:
"£45.00" is already formatted value, suggested to pass value in a raw state, and format it inside template.
For this purpose you can add global filter for vue app "currency" using http://openexchangerates.github.io/accounting.js/

few examples of how it will work:
`{{45.00 | currency}} => £45.00`
`{{45 | currency}} => £45.00`
`{{45.45464 | currency}} => £45.45`

Reply:
@coremyslo thanks, I will transfer this suggestion to Vue implementation of this component